docs/StyleCustomization.md

```diff
@@ -0,0 +1,67 @@
+# Style Customization
+
+By default Hydrogen leans towards plain styling so that its elements can be customized easily to your liking.
+
+Below are some tips to get started with styling Hydrogen. If you are unfamiliar with this topic there is more detailed information availible in the [Atom Flight Manual](http://flight-manual.atom.io/using-atom/sections/basic-customization/#style-tweaks).
+
+Start out by opening your styles.less file or searching your command palette for "stylesheet", you will see atom's: **Application: Open Your Stylesheet** command.
+
+To follow these examples make sure you have theme variables imported by puting this at the top of your styles.less file.
+
+```less
+@import "ui-variables";
+@import "syntax-variables";
+
+```
+
+In the example below there are color variables like `@my-color-variable` used. Of course, you can change these as desired.
+
+## Style Examples
+### Multiline Result View
+
+Change the result view background color for multi-line output:
+
+In your styles.less:
+
+```less
+.hydrogen {
+  .multiline-container {
+    .cell_display {
+      background-color: @background-color-highlight;
+    }
+  }
+}
+```
+
+
+Change the icon color when you hover over the toolbar icons:
+
+```less
+.hydrogen {
+  .multiline-container {
+    .toolbar .icon:hover {
+      background-color: @background-color-selected;
+    }
+  }
+}
+```
+
+### Importing Hydrogen Style Variables
+
+If you want to use Hydrogen's style variables you can do so by adding this to your styles.less:
+
+```less
+  @import "./packages/Hydrogen/styles/hydrogen.less";
+```
+
+Once you've done this you will have access to any style variables within the Hydrogen package:
+
+```less
+@hydrogen-light-background
```

Comment:
I think you could redefine them either way.

I thought about suggesting that you save your hydrogen stuff outside your styles.less then import it to avoid clutter (I do this for various style then comment in/out as needed). But I left out for now for simplicity.

Save a file of stuff in the `~/.atom/hydrogen-custom-styles.less` then in your styles.less:

```less
@import "hydrogen-custom-styles.less";
```